app.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dahlia-ah-backend",
+  "scripts": {},
+  "env": {
+    "HEROKU_APP_NAME": {
+      "required": true
+    },
+    "HEROKU_PARENT_APP_NAME": {
+      "required": true
+    }
+  },
+  "formation": {
+    "web": {
+      "quantity": 1
+    }
+  },
+  "addons": [
+    "heroku-postgresql"
+  ],
+  "buildpacks": [
+    {
+      "url": "heroku/nodejs"
+    }
+  ],
+  "stack": "heroku-18"
+}
```

Comment:
New line required at the end of file

Reply:
@allebd is there a way we can configure ESLint to automatically flag "no new line" as an error?
This issue is reoccurring too many times.

Reply:
It only happens on non javascript files. It works fine and flags it if its a javascript file

Reply:
This rule has to be added to .eslintrc
"eol-last": ["error", "always"]

Reply:
@allebd Changes have been made

Reply:
> It only happens on non javascript files. It works fine and flags it if its a javascript file

So it should also flag JSON file since it's essentially JavaScript. I'm sure though

Reply:
> This rule has to be added to .eslintrc to flag the error
> "eol-last": ["error", "always"]
> or for other options view https://eslint.org/docs/rules/eol-last

We'll work on adding that. Thanks.